migrations/sql/V1.35__add_user_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE minespace_users (
+    id serial PRIMARY KEY,
```

Comment:
Please rename this column to "user_id" so that it matches the foreign key name we plan to use.

Reply:
changed